source/simulator/parameters.cc

```diff
@@ -381,6 +389,27 @@ namespace aspect
                            "Turns on extra information on the AMG solver. Note that this will generate much more output.");
       }
       prm.leave_subsection ();
+      prm.enter_subsection ("Operator splitting parameters");
+      {
+        prm.declare_entry ("Reaction time step", "1000.0",
+                           Patterns::Double (0),
+                           "Set a time step size for computing reactions of compositional fields and the "
+                           "temperature field in case operator splitting is used. This is only used "
+                           "when the nonlinear solver scheme ``operator splitting'' is selected. "
+                           "Units: Years or seconds, depending on the ``Use years "
```

Comment:
What should I pick if I want to disable this and only use the number of steps below? 0? infinity? Document that.